Description: Fix privacy breach.
 Do not link to a non-local image.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-03-10

--- python-semantic-version-2.3.1.orig/README.rst
+++ python-semantic-version-2.3.1/README.rst
@@ -9,9 +9,6 @@ python-semanticversion
 This small python library provides a few tools to handle `SemVer`_ in Python.
 It follows strictly the 2.0.0 version of the SemVer scheme.
 
-.. image:: https://secure.travis-ci.org/rbarrois/python-semanticversion.png?branch=master
-    :target: http://travis-ci.org/rbarrois/python-semanticversion/
-
 semantic_version supports Python 2.6, 2.7, 3.2, 3.3, and is distributed under the two-clause BSD licence.
 
 Links
